### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matthewtrask**
added a `converters` definition since that one always confused me :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ Conceptually contract testing is the same thing as [data validation](#data-valid
 
 There is also client-side contract testing, where you write out the contract in a client test suite then make calls to the API to see if it matches.
 
+## Converter
+
+A set of tools that allow you to convert the OpenAPI Specification file to another API format such as API Blueprint, HAR, RAML, and others. Certain convereter tools also let you convert your API documentation from RAML for example to the OpenAPI format.
```
Comment (philsturgeon) [inline README.md]:
```suggestion
A set of tools that allow you to convert OpenAPI descriptions to another description format, such as API Blueprint, HAR, RAML, etc., or vice versa! Some will also convert to HTML which is usually for making [reference documentation](#reference-documentation).
```

Comment (philsturgeon) [reply README.md]: How's about this @matthewtrask

**Review by philsturgeon — CHANGES_REQUESTED** [review]

**Comment by matthewtrask** [comment]
@philsturgeon that looks good!

**Comment by matthewtrask** [comment]
just committed it up, so this should be good @philsturgeon

**Review by cebe — APPROVED** [review]

**Review by philsturgeon — APPROVED** [review]

**Pull request merged**